dask_deltatable/types.py

```diff
@@ -0,0 +1,6 @@
+from __future__ import annotations
+
+from typing import Any, List, Tuple, Union
+
+Filter = Tuple[str, str, Any]
```

Comment:
does this work with `object` instead of `Any`?
```suggestion
Filter = Tuple[str, str, object]
```

Reply:
mypy doesn't seem happy:

```
dask_deltatable/utils.py:55: error: "object" object is not iterable  [misc]
dask_deltatable/utils.py:56: error: Cannot determine type of "col"  [has-type]
dask_deltatable/utils.py:57: error: Cannot determine type of "op"  [has-type]
dask_deltatable/utils.py:58: error: Cannot determine type of "col"  [has-type]
dask_deltatable/utils.py:58: error: Cannot determine type of "val"  [has-type]
Found 5 errors in 1 file (checked 7 source files)
```